.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
+
+language: php
+
+php:
+  - 7.1
+  - nightly
+
+env:
+  matrix:
+    - DEPENDENCIES="high"
+    - DEPENDENCIES="low"
+
+before_install:
+  - sudo add-apt-repository ppa:git-core/ppa -y
+  - sudo apt-get update -q
+  - sudo apt-get install -y --only-upgrade git
+
+before_script:
+  - sh .travis.install.sh
+
+script:
+  - ./vendor/bin/phpunit --disallow-test-output --report-useless-tests --coverage-clover ./clover.xml
```

Comment:
`--disallow-test-output --report-useless-tests` can be set on the PHPUnit config file